app/routes/companyInterest/components/CompanyInterestList.tsx

```diff
@@ -233,11 +233,7 @@ class CompanyInterestList extends Component<Props, State> {
             </a>
           ) : (
             <Tooltip
-              style={
-                this.props.selectedSemesterOption.year
-                  ? { display: 'none' }
-                  : undefined
-              }
+              disabled={this.props.selectedSemesterOption.id != 0}
```

Comment:
```suggestion
              disabled={!!this.props.selectedSemesterOption.id}
```
To match the one underneath